src/test-run/commands/actions.js

```diff
@@ -76,7 +76,16 @@ function initDialogHandler (name, val, { skipVisibilityCheck, testRun }) {
         builder = new ClientFunctionBuilder(fn, options, { instantiation: methodName, execution: methodName });
 
     return builder.getCommand([]);
+}
+
+function initSwitchToWindowHandler (name, val) {
```

Comment:
`createSwitchToWindowHandler(name, val)`

Reply:
I call it `init` because we already have `initDialogHandler`. And here the argument name is `init` as well:
`{ name: 'findWindow', init: initSwitchToWindowHandler, required: true }`

Reply:
Ok